backend/config/environments/production.rb

```diff
@@ -15,8 +15,8 @@
   # Full error reports are disabled and caching is turned on.
   config.consider_all_requests_local = false
   config.action_controller.perform_caching = true
-  # TODO: fix this later, directupload is checkin it on staging but we want to allow non https localhost requests
-  config.action_controller.forgery_protection_origin_check = false
+  # we want to allow non https localhost requests to staging, but not production
+  config.action_controller.forgery_protection_origin_check = (ENV["INSTANCE_ROLE"].present? && ENV["INSTANCE_ROLE"] == "production") || false
```

Comment:
hmm, it's ok, but maybe a little less readable than just setting this as false if `ENV["INSTANCE_ROLE"] == "staging"`. The default value for that setting is true, and I bet it won't change in future rails versions.

Reply:
right, I'll change that, thanks!